src/main/webapp/scripts/index-script.js

```diff
@@ -0,0 +1,13 @@
+
+    var dataPromise = updateAccountData();
+
+    dataPromise.then(function(){
+        loadNav();
+        console.log("logged in 2"+loggedInStatus);
```

Comment:
Similar comment to above. Probably not necessary to have console.log()

Reply:
removed both